fieldservice_account_analytic/models/fsm_order.py

```diff
@@ -58,6 +58,6 @@ def _onchange_customer_id_location(self):
     def write(self, vals):
         res = super(FSMOrder, self).write(vals)
         for order in self:
-            if "customer_id" not in vals and order.customer_id is False:
-                order.customer_id = order.location_id.customer_id.id
+            if "customer_id" not in vals and not order.customer_id:
+                order.customer_id = order.location_id.customer_id
```

Comment:
@michelerusti I meant something like this:

```
    def write(self, vals):
        if "customer_id" not in vals and order.customer_id is False:
            vals.update({"customer_id": self.location_id.customer_id.id})
        res = super(FSMOrder, self).write(vals)
```

Reply:
@max3903 I've edited,  me know if it's alright!